runtime/moonriver/src/lib.rs

```diff
@@ -132,6 +135,7 @@ pub mod currency {
 
 	pub const TRANSACTION_BYTE_FEE: Balance = 10 * MICROMOVR * SUPPLY_FACTOR;
 	pub const STORAGE_BYTE_FEE: Balance = 100 * MICROMOVR * SUPPLY_FACTOR;
+	pub const WEIGHT_FEE: Balance = 50 * KILOWEI * SUPPLY_FACTOR;
```

Comment:
We should do some basic testing to ensure this is reasonable for your use case. It appears to be taken from #1218, which is still a WIP.

Reply:
You are right in the sense that a token transfer through XCM would cost 50_000_000_000_000, or essentially, 50microMOVR. Maybe this is too low since for KSM we are charging 0.1KSM. I will increase it to match it

Reply:
I increased it to 100KILOWEI. This means that for 1e9 weight (roughly, the amount needed for a token transfer in XCM) we charge 0.1milliMOVR, which I think gets us closer to what we charge for other assets